src/components/Routes/ClientRoutes.js

```diff
@@ -1,15 +1,29 @@
-import React from "react";
-import { Routes, Route } from "react-router-dom";
-import Wallet from "../../pages/Wallet/Wallet";
-import TransferStatus from "../../pages/TransferStatus/TransferStatus";
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import SendTokens from '../../pages/SendTokens/SendTokens';
+import TransferStatus from '../../pages/TransferStatus/TransferStatus';
+import Wallet from '../../pages/Wallet/Wallet';
 
 const ClientRoutes = () => {
-  return (

```

Comment:
Please, fix merge conflicts with main brunch

Reply:
I have added a "Send Tokens" route, and also a test function for that route on ClientRoutes.test.js. Everything else I kept the same.